packages/bahmni-widgets/src/search/commonSearch/CommonSearchWidget.tsx

```
@@ -172,9 +172,7 @@ const CommonSearchWidget = ({ extensionParams }: SearchWidgetProps) => {
       />
     );
 
-  const privilegedContexts = config.filter((c) =>
-    hasPrivilege(userPrivileges ?? null, c.requiredPrivileges),
-  );
+  const privilegedContexts = processContextConfigs(config, userPrivileges ?? null);
```

ESLint

[error] Replace `config,·userPrivileges·??·null` with `⏎····config,⏎····userPrivileges·??·null,⏎··`

(prettier/prettier)